src/wmnds/components/content-tiles/related-content/_related-content.scss

```diff
@@ -0,0 +1,12 @@
+.wmnds-content-tile__copy {
```

Comment:
Not sure why this is copy?
The top class should not have double underscores.

And the children elements also don't conform to BEM standards.